<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planificador de Tareas</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLabel" name="mainTitle">
     <property name="text">
      <string>ðŸ”± VYMTIA ðŸ”±</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="subTitle">
     <property name="text">
      <string>Planificador de Tareas</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="promptTextEdit">
     <property name="placeholderText">
      <string>Ingrese su prompt aquÃ­...</string>
     </property>
     <property name="minimumHeight">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Generar Tareas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportCsvButton">
       <property name="text">
        <string>Exportar a CSV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportPdfButton">
       <property name="text">
        <string>Exportar a PDF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="logoutButton">
       <property name="text">
        <string>Cerrar SesiÃ³n</string>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="outputLabel">
     <property name="text">
      <string>Esperando su prompt...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeft</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="resultTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>AcÃ¡ se mostrarÃ¡n las tareas generadas...</string>
     </property>
     <property name="minimumHeight">
      <number>250</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
